initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -100,21 +100,35 @@ if [ -n "${lpath}" -a -h "${lpath}" ] ; then
 fi
 
 if [ -e /etc/debian_version ]; then
-    echo "Removing DEB"
-    apt-get remove -y hpccsystems-platform
-    if [ $? -ne 0 ]; then
-       echo "An error has occured during removal.  Try \`apt-get -f remove hpccsystems-platform\` to fix this issue."
-       exit 1
+    pckname=$(apt-cache pkgnames hpccsystems-platform 2> /dev/null)
+    if [ -n "${pckname}" ] ; then
+        echo "Removing ${pckname}"
+        apt-get remove -y ${pckname}
```

Comment:
need to purge to remove package name left behind after removal
dpkg -P ${pckname}